app/MMB/src/scenes/timeline/event/IconVerticalLine.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Icon, StyleSheet, Color } from '@kiwicom/mobile-shared';
+
+type IconType = {|
+  name: string,
+  type: 'classic' | 'special' | 'ongoing',
+|};
+
+export type IconVerticalLineType = {|
+  icon: IconType,
+|};
+
+type Props = {|
+  ...IconVerticalLineType,
+  topOffset: number,
+|};
+
+const styles = StyleSheet.create({
+  iconVerticalLine: {
+    alignItems: 'center',
+    width: 30,
+    marginStart: 5,
+    marginEnd: 10,
+  },
+  topLine: {
+    width: 1,
+    backgroundColor: 'grey',
+  },
+  bottomLine: { width: 1, flex: 1, backgroundColor: 'grey' },
```

Comment:
Please, use colors from the shared package. These borders and backgrounds (or brand colors) are changing from time to time.